src/xtgeo/grid3d/_roff_grid.py

```diff
@@ -364,7 +373,7 @@ def to_file(self, filelike, roff_format=roffio.Format.BINARY):
         roffio.write(filelike, data, roff_format=roff_format)
 
     @staticmethod
-    def from_file(filelike):
+    def from_file(filelike: IO) -> RoffGrid:
```

Comment:
I cant find any closures for this filelike object, this can lead some bugs if the closure only happens when the refcount is zero. I think we should create a task for this for further investigations.

Reply:
I believe filelike should be `Union[str, pathlib.Path, io.BytesIO, io.StringIO]`

Reply:
> I believe filelike should be `Union[str, pathlib.Path, io.BytesIO, io.StringIO]`

Yeah, i think your right.